src/openms/source/ANALYSIS/OPENSWATH/SpectrumExtractor.cpp

```diff
@@ -211,7 +245,7 @@ namespace OpenMS
   {
     params.clear();
 
-    params.setValue("rt_window", 30, "Retention time window in seconds.");
+    params.setValue("rt_window", 30.0, "Retention time window in seconds.");
 
     params.setValue("min_score", 0.7, "The minimum score a spectrum must have to be assignable to a transition.");
```

Comment:
I would have to look more into the data to see what the a good default would be.  It would also be useful just to better understand what the typical scores looks like.  We should sit down at some point and go over a full example with scores and visualizations of the peaks.  This is not required now, but will be useful for actual use of the tool.